src/ir/enum_ty.rs

```diff
@@ -71,16 +71,20 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
+            let val = if is_signed {
+                cursor.enum_val_signed().map(|val| EnumVariantValue::Signed(val))
```

Comment:
Just for reference, you can do `cursor.enum_val_signed().map(EnumVariantValue::Signed)` :)